project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject clanhr/file-management "0.7.1"
+(defproject clanhr/file-management "0.7.2"
```

Comment:
You have on the message ` Now the put-file function returns immediately`, should be the async-put-file?

Also, should be 0.8.0 version? You didn't solve a but, you added some functionality. Or even 1.0.0 if the new result return is breaking,